store/ethereum.go

```diff
@@ -2,49 +2,62 @@ package store
 
 import (
 	"context"
-	"time"
 
 	sq "github.com/Masterminds/squirrel"
 )
 
-type IncomingEthereumTransaction struct {
-	Hash               string    `db:"hash"`
-	ValueWei           string    `db:"value_wei"`
-	StellarAddress     string    `db:"stellar_address"`
-	WithdrawExpiration time.Time `db:"withdraw_expiration"`
-	Withdrawn          bool      `db:"withdrawn"`
+type EthereumDeposit struct {
+	// ID is the globally unique id for this deposit
+	ID string `db:"id"`
+	// Token is the address (0x0 in the case that eth was deposited)
+	// of the tokens which were deposited to the bridge
+	Token string `db:"token"`
+	// Sender is the address of the account which deposited the tokens
+	Sender string `db:"sender"`
+	// Destination is the intended recipient of the bridge transfer
+	Destination string `db:"destination"`
+	// Amount is the amount of tokens which were deposited to the bridge
+	// contract
+	Amount string `db:"amount"`
+	// Hash is the hash of the transaction containing the deposit
+	Hash string `db:"hash"`
+	// LogIndex is the log index within the ethereum block of the deposit event
+	// emitted by the bridge contract
+	LogIndex uint `db:"log_index"`
+	// BlockNumber is the sequence number of the block containing the deposit
+	// transaction
+	BlockNumber uint64 `db:"block_number"`
+	// Timestamp is the unix timestamp of the deposit
+	Timestamp int64 `db:"block_time"`
```

Comment:
Nit: all other names are corresponding to `db` tag.
```suggestion
	BlockTime int64 `db:"block_time"`
```